src/_pyinstaller_hooks_contrib/tests/test_libraries.py

```diff
@@ -1235,3 +1235,14 @@ def test_pyshark(pyi_builder):
         #print(data)
         """
     )
+
+
+@importorskip('pyqtgraph')
+def test_pyqtgraph(pyi_builder):
+    pyi_builder.test_source(
+        """
+        import pyqtgraph.graphicsItems.PlotItem
```

Comment:
Those modules have dynamic imports for the template modules that the hook declares as hidden. 👍